rerun_py/pyproject.toml

```diff
@@ -14,7 +14,7 @@ classifiers = [
 dependencies = [
   # Must match list in `.github/workflows/reusable_test_wheels.yml`
   "attrs>=23.1.0",
-  "numpy>=1.23,<2",
+  "numpy>=1.23",
```

Comment:
This invalidates the `pixi.lock` file, which explains the CI failure. Could you run any pixi command (e.g. `pixi run rerun`) to update the lock file and push it?

Reply:
Done.